accounts/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class UserInfo(models.Model):
+	"""
+	A Separate class to store additional information 
+	about the user.
+	
+	"""
+	user = models.OneToOneField(User, on_delete=models.CASCADE)
+	email = models.EmailField(unique=True)
```

Comment:
Why do we have email again? I can always do

`lumos_user.id.email` where `id` is the Django `User` object